lib/datasources/SparqlDatasource.js

```diff
@@ -73,22 +73,31 @@ SparqlDatasource.prototype._getPatternCount = function (sparqlPattern, callback)
   // Execute the count query
   var countResponse = this._request({
     url: this._endpointUrl + encodeURIComponent(this._createCountQuery(sparqlPattern)),
-    headers: { accept: 'text/turtle;q=1.0,text/ntriples;q=0.5,text/n3;q=0.3' },
+    //This should be one of the formats supported by http://www.w3.org/TR/sparql11-overview/#sparql11-results
+    //Default to the CSV type
+    headers: { accept: 'text/csv;q=1.0,application/sparql-results+xml;q=0.5,application/sparql-results+json;q=0.3'},
```

Comment:
There's probably no point yet in accepting anything else than CSV, right? Is this supported by most implementations?